fast_llm/models/gpt/model.py

```diff
@@ -71,6 +75,30 @@ def __init__(
         else:
             self._flash_varlen_preprocessor = FlashAttnVarlenPreprocessor(self._config.transformer, self._tensor_space)
 
+    def get_language_model_layers(self) -> list[Layer]:
+        if self._config.num_multi_token_prediction_heads:
+            return [
+                layer
+                for i in range(self._config.num_multi_token_prediction_heads)
+                for layer in [
+                    MultiTokenPredictionTransformerLayer(
```

Comment:
The last one can be a plain `TransformerLayer`, no need for stacking. And if the next token is predicted last, it can be followed by a plain `LanguageModelHead`